calico/network-policy/get-started/kubernetes-default-deny.mdx

```diff
@@ -100,7 +100,7 @@ kind: GlobalNetworkPolicy
 metadata:
   name: deny-app-policy
 spec:
-  namespaceSelector: has(projectcalico.org/name) && projectcalico.org/name not in {"kube-system", "calico-system", "tigera-system"}
+  namespaceSelector: kubernetes.io/metadata.name not in {"calico-system", "kube-public", "kube-system", "tigera-operator", "tigera-system"}
```

Comment:
tigera-system is an enterprise and cloud only namespace, so not needed for OSS docs

Reply:
Thanks. I've made the general changes in all products and versions, and removed "tigera-system" from all OSS versions.